app/404.html

```diff
@@ -0,0 +1,52 @@
+<html>
+<head>
+  <title>MetaMask</title>
+  <style>
+    *{
+        padding: 0;
+        margin: 0;
+        box-sizing: border-box;
+    }
+    img{
+        display: block;
+    }
+    html, body{
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        width: 100%;
+        height: 100%;
+    }
+    .app{
+        position: relative;
+        width: 100%;
+        height: auto;
+        overflow: hidden;
+    }
+    img{
+        display: block;
+        width: 100%;
+        height: auto;
+    }
+    h2{
+        display: block;
+        width: 100%;
+        overflow: hidden;
+        position: absolute;
+        bottom: 20%;
+        left: 0;
+        color: #1b243d;
+        text-align: center;
+    }
+    h2 > a{
+        color: #1b243d;
+    }
+  </style>
+</head>
+<body>
+    <div class="app">
+        <img src="./images/404.png" alt="">
+        <h2>Powered by <a href="https://www.portal.network/">Portal Network</a></h2>
```

Comment:
I wasn't sure how I felt about this at first, but you know, getting credit for what you do seems pretty fair. I'm fine leaving it up, at least for a while. The same way we link to 409H for the phishing detector.